src/main/kotlin/com/exactpro/th2/lwdataprovider/configuration/Configuration.kt

```diff
@@ -74,6 +74,9 @@ class Configuration(customConfiguration: CustomConfigurationClass) {
         require(grpcBackPressureReadinessTimeoutMls > 0) {
             "grpcBackPressureReadinessTimeoutMls ($grpcBackPressureReadinessTimeoutMls) must be positive"
         }
+        if (execThreadPoolSize == 1) {
```

Comment:
Maybe it isn't actual now, but thread pool size shouldn't limit number of client requests which components can be handle at the same time